tests/Collection/CollectionFunctionalTest.php

```diff
@@ -119,6 +120,21 @@ public function testDropIndexShouldNotAllowWildcardCharacter()
         $this->collection->dropIndex('*');
     }
 
+    public function testExplain()
+    {
+        if (version_compare($this->getServerVersion(), '3.0.0', '<')) {
+            $this->markTestSkipped('The explain option requires server version >= 3.0');
```

Comment:
This reads like it was copied from an aggregation test. Let's copy the ExplainFunctionalTest message instead:

```php
$this->markTestSkipped('Explain command is not supported');
```